test/e2e/v2/e2e_suite.go

```diff
@@ -32,36 +32,66 @@ import (
 )
 
 var _ = Describe("kubebuilder", func() {
+
+	BeforeSuite(func() {
+		k := &utils.Kubectl{}
+
+		By("installing cert manager bundle")
+		Expect(k.InstallCertManager()).To(Succeed())
+
+		By("installing prometheus operator")
+		Expect(k.InstallPrometheusOperManager()).To(Succeed())
```

Comment:
kubectl.InstallCertManager -> shows not accurate. 
the Install Cert should be part of the kubebuilder test helpers.